web/src/components/filter/CameraGroupSelector.tsx

```diff
@@ -556,34 +559,38 @@ export function CameraGroupEdit({
         />
 
         <Separator className="flex my-2 bg-secondary" />
-        <FormField
-          control={form.control}
-          name="cameras"
-          render={({ field }) => (
-            <FormItem>
-              <FormLabel>Cameras</FormLabel>
-              <FormDescription>Select cameras for this group.</FormDescription>
-              {[
-                ...(birdseyeConfig?.enabled ? ["birdseye"] : []),
-                ...Object.keys(config?.cameras ?? {}),
-              ].map((camera) => (
-                <FormControl key={camera}>
-                  <FilterSwitch
-                    isChecked={field.value && field.value.includes(camera)}
-                    label={camera.replaceAll("_", " ")}
-                    onCheckedChange={(checked) => {
-                      const updatedCameras = checked
-                        ? [...(field.value || []), camera]
-                        : (field.value || []).filter((c) => c !== camera);
-                      form.setValue("cameras", updatedCameras);
-                    }}
-                  />
-                </FormControl>
-              ))}
-              <FormMessage />
-            </FormItem>
-          )}
-        />
+        <div className="max-h-[40vh] overflow-y-auto">
```

Comment:
maybe want to do `dvh` here instead of `vh` so it adapts when rotating a device?